src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+  --pf-c-sequence__number--m-current--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--m-current--MaxHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--m-current--BorderColor: var(--pf-global--primary-color--100);
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+  background-color: var(--pf-c-sequence--BackgroundColor);
+
+  // Hide scrollbars
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.pf-c-sequence__control {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  padding:
+    var(--pf-c-sequence__control--PaddingTop)
+    var(--pf-c-sequence__control--PaddingRight)
+    var(--pf-c-sequence__control--PaddingBottom)
+    var(--pf-c-sequence__control--PaddingLeft);
+  border-top: var(--pf-c-sequence__control--BorderTopHeight) solid var(--pf-c-sequence__control--BorderTopColor);
+  border-bottom: var(--pf-c-sequence__control--BorderBottomHeight) solid var(--pf-c-sequence__control--BorderBottomColor);
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopHeight) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomHeight) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-top: var(--pf-c-sequence__control--active--BorderTopHeight) solid var(--pf-c-sequence__control--active--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomHeight) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopHeight) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomHeight) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+}
+
+.pf-c-sequence__control-left {
+  left: var(--pf-c-sequence__control-left--PositionLeft);
+  background-color: var(--pf-c-sequence__control-left--BackgroundColor);
+  border-right: var(--pf-c-sequence__control-left--BorderRightWidth) solid var(--pf-c-sequence__control-left--BorderRightColor);
+}
+
+.pf-c-sequence__control-right {
+  right: var(--pf-c-sequence__control-right--PositionRight);
+  background-color: var(--pf-c-sequence__control-right--BackgroundColor);
+  border-left: var(--pf-c-sequence__control-right--BorderLeftWidth) solid var(--pf-c-sequence__control-right--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  overflow: scroll;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+
+  // added so that the list will be read by screen reader
+  &::before {
+    width: 1px;
+    content: "";
```

Comment:
I did style it like above, however it was added extra spacing and shifting the design a lot. I was hoping it would work with any content but it looks like "\200B" is necessary

Reply:
I think it would be a good discussion for the group, especially since it does affect styling and is an extra step to get working

Reply:
I added it back in, and it reads: "list three items" but only after it reads the content of the first list item

Reply:
Weird. Thanks for testing this.

Reply:
@jgiardino from the a11y discussion, Ill remove: 

ul li::before {
  content: "\200B";
}

and add in a a role for the list and for the list item?